assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -165,7 +165,8 @@ function Media3pPane(props) {
     // We display the media name if there's media to display or a category has
     // been selected.
     const shouldDisplayMediaSubheading = Boolean(
-      (state.isMediaLoaded && state.media) ||
+      state.isMediaLoaded ||
+        state?.media.length ||
```

Comment:
why did this change?

seemed like it was correct with

 Boolean(
    (state.isMediaLoaded && state.media) ||
        state.categories.selectedCategoryId	  );

Reply:
Unfortunately that leaves the situation where isMediaLoaded is false and isMediaLoading is true (while we're loading a second page and there *is* media there) during which we want to display this. This new logic better caters for the cases.